CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 
+- Updated nav styling to fit Swedbank Pay better.
```

Comment:
Is this all you have done in this branch?

Reply:
Roughly, yes. Should I outline all the changes to nav i've made? They're all visual as far as i can tell. Except maybe clicking the parent makes the child active, but this is not included functionality when someone uses the nav, it only works for our documentation.

Reply:
Is not visual changes just as important for users of the DG? 
My point is; when I read this I don't really know what changed. 
What can I expect if I update the DesignGuide? 

First part: Something changed in nav
Second: To fit Swedbank Pay, is this not the reason for almost all our changes?

Reply:
Yes that is the reason for all our changes, but the nav hasn't really been handled so far in the rebranding. I'm just not entirely sure what you want me to include in the docs. We should discuss this face to face aswell.